packages/graphql/src/types/app.ts

```diff
@@ -38,7 +38,23 @@ export type QuestionResponseValue = {
   value: string
 }
 
-export type QuestionResponse = QuestionResponseChoices | QuestionResponseValue
+export type QuestionResponseFlashcard = {
+  correctness: number
+}
+
+export type AggregatedResponse = AggregatedResponseFlashcard
```

Comment:
what could be aggregated responses for the other types? we need to think about this when starting with learning analytics